inst/octsympy_tests.m

```diff
@@ -18,22 +18,37 @@
 
 %% -*- texinfo -*-
 %% @defun octsympy_tests ()
-%% Run the OctSymPy tests, log results, and return true if any fail.
+%% Run the test suite, log results, and return true if any fail.
 %%
-%% I threw this together by modifying "__run_test_suite__.m" which
-%% is Copyright (C) 2005-2013 David Bateman and part of GNU Octave,
-%% GPL v3.
+%% On recent Octave, this is a thin layer around the built-in Octave
+%% command @code{__run_test_suite (@{'.'@}, @{@})}.
 %%
-%% FIXME: once we no longer try to support Octave 3.6, drop most of
-%% this and call "__run_test_suite__(@{'@@sym', '@@symfun'@}, @{@})"
-%% instead.  See https://savannah.gnu.org/bugs/?41215
+%% Testing should work once the package is installed, which is otherwise
+%% harder to test (need to know the installation directory).
 %%
+%% TODO: eventually we should drop this file altogether, but then how
+%% do we test the installed package?  Perhaps we could keep this here
+%% until @code{pkg test} works upstream: @url{https://savannah.gnu.org/bugs/?41215}
+%%
+%% @seealso{test, runtests, doctest}
 %% @end defun
 
-%% Author: Colin B. Macdonald, David Bateman
-%% Keywords: tests
-
 function anyfail = octsympy_tests ()
+  if (compare_versions (OCTAVE_VERSION (), '4.2.1', '>='))
```

Comment:
This should be '4.4.0' now (and corresponding comments) due to https://savannah.gnu.org/bugs/?51660

Reply:
thanks!  I'll look into merging this...